### ISSUE TITLE
U7 ADA Fixes

### ISSUE THREAD

**Issue by lmock** [bjc-edc-2017-18/bjc-r#13]
- [x] U7L1P1 - Missing Alt Text (Line 80)
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/7-recursion/1-trees/1-recursive-tree.html
`<img class="indent" src="/bjc-r/img/7-recursion/tree5.png" alt="" title="tree5 result: Tree with trunk and branches" title="tree5 result: Tree with trunk and branches" /> 
`

- [x] U7L1 P2 - Table missing headers, caption, scope
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/7-recursion/1-trees/2-base-case.html

- [x] U7L1 P2 - More descriptive link back to the Unit (Line 140)
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/7-recursion/1-trees/2-base-case.html
`<div class="endnote">This is an example of exponential growth. This kind of growth for an algorithm is considered to take <a href="/bjc-r/cur/programming/5-algorithms/3-timing-experiments/3-classifying-algorithms.html?topic=nyc_bjc%2F5-algorithms.topic&course=bjc4nyc.html&novideo&noassignment" title="Classifying Algorithms"><em>unreasonable time</em></a>. This is why we suggested <em>against</em> trying 20 levels. </div>
`

- [x] U7L1 P4 - file name has a typo (Line 202)
https://github.com/bjc-edc-2017-18/bjc-r/blob/gh-pages/cur/programming/7-recursion/1-trees/4-tree-variations-self-check.html
`<img class="imageRight noshadow" src="/bjc-r/img/7-recursion/tree-variation-right-angle-brnaches.jpg" alt="Tree with right angle branches" title="Tree with right angle branches" />
`

**Comment by maryfries** [comment]
I will do the tables stuff: U7L1 P2 - Table missing headers, caption, scope

**Comment by maryfries** [comment]
U7L1 P2 - I skipped the caption. To be discussed. @lmock
